docs/demo_api.rst

```diff
@@ -14,6 +17,16 @@ sample API documentation and generated content
       :private-members:
       :special-members:
 
+.. rst-example:: Python function with multiple signatures
+
+   .. py:function:: func(a: int, b: int) -> int
+                    func(a: float, b: float) -> float
```

Comment:
The viewcode extension is trying to hyperlink to the source here, but it just leads to the 
![image](https://user-images.githubusercontent.com/14963867/166116996-337f770c-6b40-4772-8db1-6074aa31ddfc.png)

Reply:
Oh, I see I accidentally reused the same name.   Fixed now.  Seems like the prior `automodule` directive is causing this other function to be considered within the `test_py_module.test` module rather than just in `test_py_module`, but that doesn't really matter too much.